docs/maintain/maintain-guides-how-to-nominate-polkadot.md

```diff
@@ -40,143 +41,42 @@ performs due diligence on the validators that they elect. When looking for valid
 nominator should pay attention to their own reward percentage for nominating a specific validator -
 as well as the risk that they bear of being slashed if the validator gets slashed.
 
-If you are a beginner, please watch the video below for detailed instructions
+If you are a beginner, please watch the video below for detailed instructions.
 
-[![Stake on Polkadot/Kusama](https://img.youtube.com/vi/FCXC0CDhyS4/0.jpg)](https://youtu.be/FCXC0CDhyS4)
+[![Stake on Polkadot/Kusama](https://img.youtube.com/vi/KTBUOFq0KO4/0.jpg)](https://youtu.be/KTBUOFq0KO4)
```

Comment:
I changed it to Ross's tutorial. Remember to change it to your tutorial after it is published @filippoweb3

Reply:
Done